ecosystem/web-wallet/src/core/components/NetworkListItem.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  Box,
+  Center,
+  Spinner,
+  Text,
+  useColorMode,
+  useRadio,
+  UseRadioProps,
+} from '@chakra-ui/react';
+import { LOCAL_NODE_URL } from 'core/constants';
+import {
+  AptosNetwork,
+  networkUriMap,
+} from 'core/utils/network';
+import React from 'react';
+
+export interface SettingsListItemProps {
+  title?: 'Mainnet' | 'Testnet' | 'Devnet' | 'Localhost';
```

Comment:
Seems like this is repeated from NetworkPreference, it should use a shared type (e.g. the `networkUriMap`).